open_city_profile/tests/snapshots/snap_test_graphql_api_schema.py

```diff
@@ -182,18 +182,25 @@
 }
 
 type DeleteMyProfileMutationPayload {
+  results: [DeletionResultNode]
   clientMutationId: String
 }
 
-input DeleteMyServiceDataMutationInput {
+input DeleteMyServiceDataInput {
```

Comment:
The name could still be `DeleteMyServiceDataMutationInput` as before. That's the naming convention used in all other mutation input root types too.